pandas/core/generic.py

```diff
@@ -52,6 +53,15 @@
 from pandas.io.formats.printing import pprint_thing
 from pandas.tseries.frequencies import to_offset
 
+if TYPE_CHECKING:
```

Comment:
We are trying to avoid using this where possible so would rather not use here. sqlite3 should be fine to actually import since its part of stdlib. sqlalchemy is not as its an optional dependency so just remove that import altogether